amplify.yml

```diff
@@ -7,12 +7,16 @@ applications:
             - nvm install 14
             - nvm use 14
             - node -v
+            - export FLUTTER_HOME=${HOME}/sdks/flutter'
```

Comment:
Note: I need to validate these changes. I am a little confused why `nvm install` is listed in preBuild and build phases. To me this implies that prebuild may not be running? If so, these will need to be added to the build phase.